lib/rubycritic/commands/default.rb

```diff
@@ -12,7 +13,12 @@ def initialize(paths)
       end
 
       def execute
-        report(critique)
+        case Config.format
+        when :json
+          report(critique)
+        else
+          open(report(critique))
+        end
```

Comment:
Since [this one](https://github.com/whitesmith/rubycritic/pull/91) is close to merge, let's update this to:

```Ruby
when :json
  report(critique)
when :html
  open(report(critique))
else
  raise "Unknown output format #{Config.format}"
end
```